pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/tilbakekreving/TilbakekrevingFerdig.kt

```diff
@@ -0,0 +1,159 @@
+package no.nav.pensjon.etterlatte.maler.tilbakekreving
+
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.Language.Nynorsk
+import no.nav.pensjon.brev.template.OutlinePhrase
+import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+import no.nav.pensjon.etterlatte.EtterlatteBrevKode
+import no.nav.pensjon.etterlatte.EtterlatteTemplate
+import no.nav.pensjon.etterlatte.maler.BrevDTO
+import no.nav.pensjon.etterlatte.maler.Element
+import no.nav.pensjon.etterlatte.maler.Hovedmal
+import no.nav.pensjon.etterlatte.maler.fraser.common.Constants
+import no.nav.pensjon.etterlatte.maler.konverterElementerTilBrevbakerformat
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingFerdigDTOSelectors.erBP
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingFerdigDTOSelectors.erOMS
+import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingFerdigDTOSelectors.innhold
+
+data class TilbakekrevingFerdigDTO(
+    override val innhold: List<Element>,
+    val erOMS: Boolean,
+    val erBP: Boolean,
+) : BrevDTO
+
+@TemplateModelHelpers
+object TilbakekrevingFerdig : EtterlatteTemplate<TilbakekrevingFerdigDTO>, Hovedmal {
+    override val kode: EtterlatteBrevKode = EtterlatteBrevKode.TILBAKEKREVING_FERDIG
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = TilbakekrevingFerdigDTO::class,
+        languages = languages(Bokmal, Nynorsk, English),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Vedtak - Tilbakekreving",
+            isSensitiv = true,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VEDTAK,
+            brevtype = LetterMetadata.Brevtype.VEDTAKSBREV,
+        ),
+    ) {
+        title {
+            text(
+                Bokmal to "Du må betale tilbake gjenlevendepensjon",
+                Nynorsk to "Du må betale tilbake gjenlevendepensjon",
+                English to "Du må betale tilbake gjenlevendepensjon"
+            )
+        }
+        outline {
+
+            konverterElementerTilBrevbakerformat(innhold)
+
+            includePhrase(DuHarRettTilAaKlage)
+            includePhrase(DuHarRettTilInnsyn)
+            includePhrase(HarDuSpoersmaal(erOMS, erBP))
+        }
+
+        // TODO EY-2806 VEDLEGG
+    }
+}
+
+private object DuHarRettTilAaKlage : OutlinePhrase<LangBokmalNynorskEnglish>() {
```

Comment:
Finnes ikke disse fra før? Eller er de helt spesifikke for tilbakekreving?

Reply:
Jo men teksten i eksempelet til Liv Inger er ulikt